### PR TITLE
remove excess memberships, and attendances/submissions/observations from instructors

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

**Comment by jshawl** [comment]
@RobertAKARobin this branch has conflicts. Also, do you mind pairing with @mattscilipoti about this pr? Nervous about the deletions

File: db/migrate/20151022183954_remove_excess.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  def change
+    Group.at_path("ga").memberships.destroy_all
+    Group.at_path("ga_wdi").memberships.destroy_all
+    Group.at_path("ga_wdi_dc").memberships.destroy_all
+
+    instructors = ["adambray", "ebirving", "andrewsunglaekim", "amaseda", "jmas13", "nolds9", "mattscilipoti", "jshawl", "robertakarobin"]
+    instructors.each do |username|
+      user = User.named(username)
+      user.attendances.destroy_all
+      user.observations.destroy_all
```
Comment (mattscilipoti) [inline db/migrate/20151022183954_remove_excess.rb]: Does this remove observations about us, or observations performed by us?

File: db/schema.rb [hunk 2]
```diff
@@ -58,8 +58,9 @@
     t.integer  "group_id"
     t.integer  "user_id"
     t.boolean  "is_admin"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+    t.boolean  "is_priority"
```
Comment (mattscilipoti) [inline db/schema.rb]: Where is the "db/schema.rb" change for this migration?

File: db/migrate/20151022183954_remove_excess.rb (hunk 1 above)
Comment (RobertAKARobin) [inline db/migrate/20151022183954_remove_excess.rb]: About us. We shouldn't have any

File: db/schema.rb (hunk 2 above)
Comment (RobertAKARobin) [inline db/schema.rb]: There isn't one, because the migration doesn't change the schema at all -- it just deletes records.

**Comment by RobertAKARobin** [comment]
Conflicts were due to previous merge. Fixed.

**Pull request closed without merging**